src/server/server_family.cc

```diff
@@ -1757,16 +1757,6 @@ void ServerFamily::ReplicaOfInternal(string_view host, string_view port_sv, Conn
 
   // Wait until all transactions finished and prevent any other from executing.
   // The --replicaof flag executed on startup has no corresponding transaction.
-  if (auto* tx = cntx->transaction; tx) {
```

Comment:
Why did you remove this ?

Reply:
Because I decided not do this functional change. I got no clear answer. The comment above it excessive (fixed)